contentcuration/contentcuration/viewsets/base.py

```diff
@@ -627,15 +627,22 @@ def perform_update(self, serializer):
     def update_from_changes(self, changes):
         errors = []
         changes_to_return = []
-        queryset = self.get_edit_queryset().order_by()
         for change in changes:
+            if 'bookmark' in change['mods'].keys():
+                try:
+                    queryset = self.get_queryset().order_by()
```

Comment:
Interesting - where was a `KeyError` being produced here?

Reply:
Also - this is dangerous, as it means that someone could edit a anything by virtue of adding a `bookmark` property to the changes.

We should localize these changes just to the `ChannelViewset` and only use this queryset for determining if we can add a bookmark - not for all changes.